### PR TITLE
Add an example of Optuna Dashboard

### PR DESCRIPTION
**Pull request by c-bata**
## Motivation

To let more users know the more advanced usage of Optuna Dashboard.
## Description of the changes

Add an example of Optuna Dashboard.

### CODE REVIEW

**Comment by c-bata** [comment]
@keisuke-umezawa Could you review this PR?

File: README.md [hunk 1]
```diff
@@ -104,7 +110,6 @@ In addition, integration modules are available for the following libraries, prov
 
 ### Examples of Distributed Optimization
 
-* [Optimizing on a Dask cluster](./dask/dask_simple.py)
```
Comment (keisuke-umezawa) [inline README.md:107]: [question] Why do you remove this one?

**Review by keisuke-umezawa — APPROVED** [review]
LGTM except one comment. You can merge this PR if the comment is trivial.

File: README.md (hunk 1 above)
Comment (c-bata) [reply README.md:107]: Oh... thank you for pointing it out. I revert it.

**Comment by c-bata** [comment]
@keisuke-umezawa Thank you for your review! PTAL.

**Review by keisuke-umezawa — APPROVED** [review]

**Pull request merged**